js/read_dir_test.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import { test, testPerm, assert, assertEqual } from "./test_util.ts";
+import * as deno from "deno";
+
+testPerm({ write: true }, function readDirSyncSuccess() {
+  const tmpDir = deno.makeTempDirSync();
+  const enc = new TextEncoder();
+  const data = enc.encode("Hello World");
+  const file1 = `${tmpDir}/hello.txt`;
+  const file2 = `${tmpDir}/world.txt`;
+  deno.writeFileSync(file1, data);
+  deno.writeFileSync(file2, data);
+  const files = deno.readDirSync(tmpDir);
```

Comment:
Rather than creating a directory of files, just do something like `readDirSync("tests/")` or `readDirSync("js/")`.
The tests are guaranteed to be run from the root of the source tree.

Reply:
Fixed.